.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I thought the `.vscode` folder should be ignored, is there a reason we want this for all the contributors?